modules/internal/ChapelArray.chpl

```diff
@@ -239,9 +239,9 @@ module ChapelArray {
     const hereID = here.id;
     const privatizeData = value.dsiGetPrivatizeData();
     on Locales[0] do
-      _newPrivatizedClassHelp(value, value, n, hereID, privatizeData);
+      _newPrivatizedClassHelp(value, value, n, hereID);
```

Comment:
just curious, when you removed `privatizeData` from the args, did this cause any commcount change? (I hope it didn't)

Reply:
It did cause comm count changes. See the commit introducing the change -- https://github.com/chapel-lang/chapel/pull/17354/commits/622c5ca731e9b0b0bdabcdf9f850897453f2d57b .

Reply:
Ah, I see why now. Thanks